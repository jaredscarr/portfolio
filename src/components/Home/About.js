import React from 'react';

import Grid from '@material-ui/core/Grid';
import Fade from '@material-ui/core/Fade';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
  root: {
    marginTop: '40vh',
    marginBottom: '20vh',
  },
  title: {
    // letterSpacing: theme.spacing(0.8),
    // fontWieght: 'bold',
  },
}));

const About = ({ menuState, onClick }) => {
  const classes = useStyles();

  return (
    <Fade in={true}>
      <Grid
        className={classes.root}
        container
        spacing={0}
        direction="column"
        justify="center"
        alignItems="center"
      >
        <Typography
          className={classes.title}
          variant="body1"
          align="center"
          paragraph
        >
          About Placeholder
        </Typography>
      </Grid>
    </Fade>
  );
}

export default About;