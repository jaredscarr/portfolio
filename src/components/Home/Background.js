import React, { useRef, useMemo, Fragment } from 'react'
import { Billboard, useTexture, Stars } from '@react-three/drei'
import { useFrame } from '@react-three/fiber'

const Cloud = ({ size = 1, opacity = 0.1, speed = 0.4, spread = 10, length = 1.5, segments = 20, dir = 1, ...props }) => {
  const group = useRef()
  const texture = useTexture("/cloud.png")

  const clouds = useMemo(
    () =>
      [...new Array(segments)].map((_, index) => ({
        x: spread / 2 - Math.random() * spread,
        y: spread / 2 - Math.random() * spread,
        scale: 0.4 + Math.sin(((index + 1) / segments) * Math.PI) * ((0.2 + Math.random()) * 10) * size,
        density: Math.max(0.2, Math.random()),
        rotation: Math.max(0.002, 0.005 * Math.random()) * speed,
      })),
    [spread, segments, speed, size],
  )

  useFrame((state) =>
    group.current?.children.forEach((cloud, index) => {
      cloud.rotation.z += clouds[index].rotation * dir
      cloud.scale.setScalar(clouds[index].scale + (((1 + Math.sin(state.clock.getElapsedTime() / 10)) / 2) * index) / 10)
    }),
  )

  return (
    <group {...props}>
      <group position={[0, 0, (segments / 2) * length]} ref={group}>
        {clouds.map(({ x, y, scale, density }, index) => (
          <Billboard key={index} scale={[scale, scale, scale]} position={[x, y, -index * length]} lockZ>
            <meshStandardMaterial map={texture} transparent opacity={(scale / 6) * density * opacity} depthTest={false} />
          </Billboard>
        ))}
      </group>
    </group>
  )
}

const Clouds = ({ menuState }) => {
  return <Cloud size={2} rotation={[0, Math.PI / 2, 0]} position={[0, 75, -100]} scale={[10, 10, 10]} />
}

const NightSky = () => {
  return ( 
    <Stars
      radius={100} // Radius of the inner sphere (default=100)
      depth={50} // Depth of area where stars should fit (default=50)
      count={1000} // Amount of stars (default=5000)
      factor={4} // Size factor (default=4)
      saturation={0} // Saturation 0-1 (default=0)
      fade // Faded dots (default=false)
    />
  )
}

const Background = ({ menuState, darkState }) => {

  const sky = darkState ? <NightSky /> : undefined

  return (
    <Fragment>
      <ambientLight />
      <pointLight position={[10, 10, 10]} />
      { sky }
      <Clouds menuState={menuState} />
    </Fragment>
  );
}

export default Background;