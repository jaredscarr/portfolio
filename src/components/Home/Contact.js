import React, { Fragment } from 'react'
import CssBaseline from '@material-ui/core/CssBaseline'
import Typography from '@material-ui/core/Typography'
import { makeStyles } from '@material-ui/core/styles'
import Container from '@material-ui/core/Container'
import Link from '@material-ui/core/Link'
import Code from '@material-ui/icons/Code'
import GitHub from '@material-ui/icons/GitHub'
import LinkedIn from '@material-ui/icons/LinkedIn'

//TODO: check this link in the copyright section and see if it works or needs adjustment. Maybe remove.
function Copyright() {
  return (
    <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <Link color="inherit" href="https://material-ui.com/">
        jaredscarr.com
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const useStyles = makeStyles((theme) => ({
  root: {
    position: 'relative',
  },
  heroContent: {
    backgroundColor: theme.palette.background.paper,
    padding: theme.spacing(8, 0, 6),
  },
  heroSubtitle: {
    marginTop: theme.spacing(10),
    letterSpacing: theme.spacing(0.8),
  },
  linkPosition: {
    marginTop: theme.spacing(5),
  },
  link: {
    margin: theme.spacing(2),
  },
  footer: {
    backgroundColor: theme.palette.background.paper,
    paddingBottom: theme.spacing(2),
  },
}));

export default function Experiments() {
  const classes = useStyles();

  return (
    <Fragment>
      <CssBaseline />
      <main className={classes.root}>
        {/* Hero unit */}
        <div className={classes.heroContent}>
          <Container maxWidth="sm">
            <Typography className={classes.heroSubtitle} variant="h6" align="center" color="textSecondary" paragraph>
              jaredscarr@gmail.com
            </Typography>
            <Typography className={classes.linkPosition} align="center">
              <Link
                color="inherit"
                variant="inherit"
                className={classes.link}
                href="https://www.linkedin.com/in/jaredscarr"
                target="_blank"
                rel="noreferrer"
              >
                <LinkedIn />
              </Link>
              <Link
                color="inherit"
                variant="inherit" 
                className={classes.link}
                href="https://codesandbox.io/u/jaredscarr"
                target="_blank"
                rel="noreferrer"
              >
                <Code />
              </Link>
              <Link
                color="inherit"
                variant="inherit"
                className={classes.link}
                href="https://github.com/jaredscarr"
                target="_blank"
                rel="noreferrer"
              >
                <GitHub />
              </Link>
            </Typography>
          </Container>
        </div>
      </main>
      {/* Footer */}
      <footer className={classes.footer}>
        <Copyright />
      </footer>
      {/* End footer */}
    </Fragment>
  );
}
