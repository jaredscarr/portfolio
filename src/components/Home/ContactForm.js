import React, { useState } from 'react'
import Button from '@material-ui/core/Button'
import Fade from '@material-ui/core/Fade'
import Grid from '@material-ui/core/Grid'
import TextField from '@material-ui/core/TextField'
import Typography from '@material-ui/core/Typography'
import { makeStyles } from '@material-ui/core/styles'

import { useFormik } from 'formik';

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
    textAlign: 'center',
    marginTop: '20vh',
    [theme.breakpoints.down('xs')]: {
      marginTop: '10vh',
    },
  },
  textFieldLayout: {
    width: '50%',
    marginBottom: theme.spacing(5),
    [theme.breakpoints.down('xs')]: {
      marginBottom: theme.spacing(2),
    },
  },
  textFieldFullWidthLayout: {
    width: '75%',
    marginBottom: theme.spacing(15),
    [theme.breakpoints.down('xs')]: {
      width: '50%',
      marginBottom: theme.spacing(5),
    },
    
  },
  submitButtonContainer: {
    marginBottom: theme.spacing(10),
    [theme.breakpoints.down('xs')]: {
      marginBottom: theme.spacing(3),
    },
  },
  buttonText: {
    fontSize: theme.spacing(1.5),
    padding: theme.spacing(0.5),
  },
  messagePlaceholder: {
    padding: theme.spacing(12),
  }
}));

const validate = values => {
  const errors = {};

  if (!values.name) {
    errors.name = 'Required';
  } else if (values.name.length > 15) {
    errors.name = 'Must be 15 characters or less';
  }

  if (!values.email) {
    errors.email = 'Required';
  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
    errors.email = 'Invalid email address';
  }

  if (!values.message) {
    errors.message = 'Required';
  } else if (values.message.length > 500) {
    errors.message = 'Must be 500 characters or less';
  }

  return errors;
};

const postData = async (url, data = {}) => {

  const response = await fetch(url, {
    method: 'POST',
    mode: 'cors',
    cache: 'no-cache',
    credentials: 'same-origin',
    headers: {
      'Content-Type': 'application/json'
    },
    redirect: 'follow',
    referrerPolicy: 'no-referrer',
    body: JSON.stringify(data),
  });
  return response.json();
}

const ContactForm = () => {
  const classes = useStyles()
  const [formVisible, setFormVisibility] = useState(true)

  const formik = useFormik({
    initialValues: {
      name: '',
      email: '',
      message: '',
    },
    validate,
    onSubmit: values => {
      setFormVisibility(false)
      const res = postData(process.env.REACT_APP_AWS_GATEWAY_URL, values)
        if (res.status === 200) {
          setFormVisibility(false)
          return res
        } else {
          return res
        }
      },
    })

  return (
    <Fade in={true}>
      <div className={classes.root}>
        {formVisible &&
        <div> 
          <form
            onSubmit={formik.handleSubmit}
            noValidate
            autoComplete="off"
          >
            <Grid
              container
              direction="row"
              alignItems="center"
            >
              <Grid item xs={12} sm={6}>
                <TextField
                  id="name"
                  className={classes.textFieldLayout}
                  label="Name"
                  onChange={formik.handleChange}
                  onBlur={formik.handleBlur}
                  value={formik.values.name}
                />
                {formik.touched.name && formik.errors.name ? (<div>{formik.errors.name}</div>) : <div><p></p></div>}
              </Grid>
              <Grid item xs={12} sm={6}>
                <TextField
                  id="email"
                  className={classes.textFieldLayout}
                  label="Email"
                  onChange={formik.handleChange}
                  onBlur={formik.handleBlur}
                  value={formik.values.email}
                />
                {formik.touched.email && formik.errors.email ? (<div>{formik.errors.email}</div>) : <div><p></p></div>}
              </Grid>
            </Grid>
            <Grid
              container
              direction="row"
              justify="flex-start"
              alignItems="center"
            >
              <Grid item xs={12}>
                <TextField
                  id="message"
                  className={classes.textFieldFullWidthLayout}
                  label="Message"
                  onChange={formik.handleChange}
                  onBlur={formik.handleBlur}
                  value={formik.values.message}
                  multiline
                  rows={2}
                  rowsMax={4}
                />
              {formik.touched.message && formik.errors.message ? (<div>{formik.errors.message}</div>) : <div><p></p></div>}
              </Grid>
            </Grid>
            <Grid
              container
              direction="row"
              justify="flex-end"
              alignItems="center"
            >
              <Grid item xs={6}>
                <div className={classes.submitButtonContainer}>
                  <Button
                    className={classes.submitButton}
                    type="submit"
                    variant="outlined"
                    size="small"
                  >
                    <Typography
                      className={classes.buttonText}
                      color="textSecondary"
                    >
                      Send
                    </Typography>
                  </Button>
                </div>
              </Grid>
            </Grid>
          </form>
        </div>
        }
        {!formVisible &&
        <div className={classes.messagePlaceholder}>
          <h5>Message Sent!</h5>
          <p>Thank you! I will get back to you as soon as possible.</p>
        </div>
        }
      </div>
    </Fade>
  );
};

export default ContactForm;