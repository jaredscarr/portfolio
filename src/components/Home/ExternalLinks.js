import React from 'react';
import { useLocation } from 'react-router-dom';

import Code from '@material-ui/icons/Code';
import GitHub from '@material-ui/icons/GitHub';
import LinkedIn from '@material-ui/icons/LinkedIn';
import { Link as MuiLink } from '@material-ui/core';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';

import getTheme from '../../Theme';


const ExternalLinks = ({ darkState }) => {
  const paletteType = darkState ? 'dark' : 'light';
  const theme = getTheme(paletteType);
  const route = useLocation();
  let background = 'inherit';

  if (route.pathname === '/projects') {
    background = darkState ? theme.palette.primary.dark : theme.palette.primary.light;
  }

  const useStyles = makeStyles((theme) => ({
    root: {
      backgroundColor: background,
      paddingTop: '10vh',
    },
    linkPosition: {
      marginTop: theme.spacing(5),
    },
    link: {
      margin: theme.spacing(2),
    },
  }));
  
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <Typography className={classes.linkPosition} align="center"> 
        <MuiLink
          color="inherit"
          variant="inherit"
          className={classes.link}
          href="https://www.linkedin.com/in/jaredscarr"
          target="_blank"
          rel="noreferrer moopener"
        >
          <LinkedIn />
        </MuiLink>
        <MuiLink
          color="inherit"
          variant="inherit" 
          className={classes.link}
          href="https://codesandbox.io/u/jaredscarr"
          target="_blank"
          rel="noreferrer moopener"
        >
          <Code />
        </MuiLink>
        <MuiLink
          color="inherit"
          variant="inherit"
          className={classes.link}
          href="https://github.com/jaredscarr"
          target="_blank"
          rel="noreferrer moopener"
        >
          <GitHub />
        </MuiLink>
      </Typography>
    </div>
  );
}

export default ExternalLinks;
