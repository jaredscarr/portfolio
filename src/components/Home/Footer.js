import React from 'react';
import Grid from '@mui/material/Grid';
import Link from '@mui/material/Link';
import Switch from '@mui/material/Switch';
import GitHub from '@mui/icons-material/GitHub';
import LinkedIn from '@mui/icons-material/LinkedIn';
import Typography from '@mui/material/Typography';
import Brightness3Icon from '@mui/icons-material/Brightness3';
import Brightness4Icon from '@mui/icons-material/Brightness4';

const Copyright = () => {
  return (
    <Typography
      fontSize={10}
      color="textSecondary"
      align="center">
      {'Copyright Â© '}
        jaredscarr.com{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const Footer = ({ darkState, handleDarkThemeChange }) => {
  // const paletteType = darkState ? 'dark' : 'light';
  let icon =  darkState ? <Brightness3Icon /> : <Brightness4Icon />;

  return (
    <footer>
      <Grid
        container
        direction="row"
        justifyContent="flex-start"
        alignItems="flex-end"
      >
        <Grid item xs={6} sm={6} md={3}>
          <Link
            color="inherit"
            variant="inherit"
            href="https://www.linkedin.com/in/jaredscarr"
            target="_blank"
            rel="noreferrer moopener"
          >
            <Typography
              align="center"
              color="textPrimary"
            > 
              <LinkedIn />
            </Typography>
          </Link>
        </Grid>
        <Grid item xs={6} sm={6} md={3}>
          <Link
            color="inherit"
            variant="inherit"
            href="https://github.com/jaredscarr"
            target="_blank"
            rel="noreferrer moopener"
          >
            <Typography
              align="center"
              color="textPrimary"
            >
              <GitHub />
            </Typography>
          </Link>
        </Grid>
        <Grid item xs={6} sm={6} md={3}>
          <Typography
            align="center"
            color="textPrimary">
            {icon}
          </Typography>
        </Grid>
        <Grid item xs={6} sm={6} md={3}>
        <Typography
          align="center"
          color="textPrimary"
        >
          <Switch checked={darkState} onChange={handleDarkThemeChange} size="small" color="secondary"/>
        </Typography>
        </Grid>
        <Grid item xs={12} sm={6} md={3}>
          <Copyright />
        </Grid>
      </Grid>
    </footer>
  );
}

export default Footer;
