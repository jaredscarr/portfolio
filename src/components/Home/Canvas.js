import React, { Suspense, useRef } from 'react';
import { Canvas } from '@react-three/fiber';

import { makeStyles } from '@material-ui/core/styles';

import Background from './Background';

const useStyles = makeStyles((theme) => ({
  container: {
    height: '100vh',
  },
}));

const BackgroundCanvas = ({ menuState, darkState }) => {
  
  const classes = useStyles();

  return (
    <div className={classes.container}>
      <Canvas className={classes.canvas} postition={[0, 0, 0]}>
        <Suspense fallback="Loading ... ">
          <Background menuState={menuState} darkState={darkState} />
        </Suspense>
      </Canvas>
    </div>
  );
}

export default BackgroundCanvas;
