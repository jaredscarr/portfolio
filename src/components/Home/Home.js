import React, { Suspense, Fragment } from "react"
import { Parallax, ParallaxLayer } from "react-spring/renderprops-addons"
import { Canvas } from "react-three-fiber"
import Background from './Background'
import Experiments from '../Projects/index'
import AppBar from '@material-ui/core/AppBar'
import Toolbar from '@material-ui/core/Toolbar'
import Typography from '@material-ui/core/Typography'
import { makeStyles } from '@material-ui/core/styles'
import Link from '@material-ui/core/Link'
import Code from '@material-ui/icons/Code'
import GitHub from '@material-ui/icons/GitHub'
import LinkedIn from '@material-ui/icons/LinkedIn'

const useStyles = makeStyles((theme) => ({
  root: {
    position: 'relative',
  },
  navroot: {
    flexGrow: 1,
    background: 'rgb(7, 7, 18)',
    color: 'grey',
  },
  navlink: {
    marginRight: theme.spacing(2),
    fontWieght: 'bold',
    letterSpacing: theme.spacing(0.4),
    color: 'grey',
    '&:hover': {
      color: 'white',
    },
  },
  title: {
    letterSpacing: theme.spacing(0.8),
    fontWieght: 'bold',
  },
  subtitle: {
    letterSpacing: theme.spacing(0.8),
  },
  linkPosition: {
    marginTop: theme.spacing(5),
  },
  link: {
    margin: theme.spacing(2),
  },
}));

const Copyright = () => {
  return (
    <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <Link color="inherit" href="https://material-ui.com/">
        jaredscarr.com
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const Home = () => {
  
  const classes = useStyles()
  let parallax;

  return (
    <Fragment>
      <AppBar position="static" className={classes.navroot}>
        <Toolbar>
          <Typography
            className={classes.navlink}
            variant="subtitle2"
            onClick={() => parallax.scrollTo(0)}
          >
            HOME
          </Typography>
          <Typography
            className={classes.navlink}
            variant="subtitle2"
            onClick={() => parallax.scrollTo(1)}
          >
            EXPERIMENTS
          </Typography>
          <Typography
            className={classes.navlink}
            variant="subtitle2"
            onClick={() => parallax.scrollTo(3)}
          >
            CONNECT
          </Typography>
        </Toolbar>
      </AppBar>
      <Parallax pages={4} ref={ref => parallax = ref}>
        <div>
          <ParallaxLayer
            offset={0}
            speed={-1.1}
            factor={0.9}
            style={{ opacity: 0.3 }}
            onClick={() => parallax.scrollTo(1)}
          >
            <Suspense fallback="Loading ... ">
              <Canvas postition={[0, 0, 0]}>
                <ambientLight />
                <pointLight position={[10, 10, 10]} />
                <Background
                  constructorArgs={[1, 8, 6, 0, Math.PI * 2, 0, Math.PI * 2]}
                  position={[0, 0, 0]}
                />
              </Canvas>
            </Suspense>
          </ParallaxLayer>
        </div>
        <ParallaxLayer
          offset={0.4}
          speed={0}
          style={{
            textAlign: "center",
            alignItems: "center",
            justifyContent: "center"
          }}
          onClick={() => parallax.scrollTo(1)}
        >
          <Typography className={classes.title} variant="h4" align="center" color="textPrimary" paragraph>
            jaredscarr.com
          </Typography>
        </ParallaxLayer>
        <ParallaxLayer offset={0.9} speed={-1}>
          <footer className={classes.footer}>
            <Copyright />
          </footer>
        </ParallaxLayer>
        <ParallaxLayer offset={1.1} speed={1} style={{ textAlign: "center" }}>
          <Experiments />
        </ParallaxLayer>
        <ParallaxLayer offset={3.4} speed={0} style={{ textAlign: "center" }}>
          <Typography className={classes.subtitle} variant="h6" align="center" color="textSecondary" paragraph>
            jaredscarr@gmail.com
          </Typography>
          <Typography className={classes.linkPosition} align="center">
            <Link
              color="inherit"
              variant="inherit"
              className={classes.link}
              href="https://www.linkedin.com/in/jaredscarr"
              target="_blank"
              rel="noreferrer moopener"
            >
              <LinkedIn />
            </Link>
            <Link
              color="inherit"
              variant="inherit" 
              className={classes.link}
              href="https://codesandbox.io/u/jaredscarr"
              target="_blank"
              rel="noreferrer moopener"
            >
              <Code />
            </Link>
            <Link
              color="inherit"
              variant="inherit"
              className={classes.link}
              href="https://github.com/jaredscarr"
              target="_blank"
              rel="noreferrer moopener"
            >
              <GitHub />
            </Link>
          </Typography>
        </ParallaxLayer>
      </Parallax>
    </Fragment>
  );
}

export default Home;
