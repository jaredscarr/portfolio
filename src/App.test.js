import React from 'react'
import 'mutationobserver-shim'
import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
import { unmountComponentAtNode } from 'react-dom'
import Home from './components/Home/Home'

global.MutationObserver = window.MutationObserver;
let container = null;
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('Home Component', () => {
  test('Home Renders correctly', () => {
    const tree = renderer.create(<Router><Home /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
  });
})

