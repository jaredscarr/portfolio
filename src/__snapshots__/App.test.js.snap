// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Home Renders correctly 1`] = `
Array [
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-navroot-2 MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root makeStyles-navlink-3 MuiTypography-subtitle2"
        onClick={[Function]}
      >
        HOME
      </h6>
      <h6
        className="MuiTypography-root makeStyles-navlink-3 MuiTypography-subtitle2"
        onClick={[Function]}
      >
        EXPERIMENTS
      </h6>
      <h6
        className="MuiTypography-root makeStyles-navlink-3 MuiTypography-subtitle2"
        onClick={[Function]}
      >
        CONNECT
      </h6>
    </div>
  </header>,
  <div
    onScroll={[Function]}
    onTouchStart={[Function]}
    onWheel={[Function]}
    style={
      Object {
        "WebkitOverflowScrolling": "touch",
        "WebkitTransform": "translate(0px,0px)",
        "height": "100%",
        "msTransform": "translate(0px,0px)",
        "overflow": "scroll",
        "overflowX": "hidden",
        "overflowY": "scroll",
        "position": "absolute",
        "transform": " translate3d(0px,0px,0px)",
        "width": "100%",
      }
    }
  />,
]
`;
